app.py

```diff
@@ -383,13 +383,13 @@ def run_optimization_cqm(run_click: int, model: str, solver: str, scenario: str,
     cancel=[Input("cancel-button", "n_clicks")],
     prevent_initial_call=True
 )
-def run_optimization_mip(run_click: int,
+def run_optimization_highs(run_click: int,
                          model: str,
                          solver: str,
                          scenario: str,
                          time_limit: int) \
                          -> tuple[go.Figure, go.Figure, str, str]:
```

Comment:
```suggestion
                           model: str,
                           solver: str,
                           scenario: str,
                           time_limit: int,
                           ) -> tuple[go.Figure, go.Figure, str, str]:
```
incredibly minor, but we should stick with a clean indent.